pcap-netfilter-linux.c

```diff
@@ -612,7 +612,7 @@ netfilter_activate(pcap_t* handle)
 
 	if (type == NFLOG) {
 		handle->linktype = DLT_NFLOG;
-		handle->dlt_list = (u_int *) malloc(sizeof(u_int) * 2);
+		handle->dlt_list = (u_int *) malloc(2 * sizeof(u_int));
 		if (handle->dlt_list != NULL) {
```

Comment:
I wonder if this should fail with an error as is done below. In any case, this change is out of scope of this commit.